src/DRAFTS/roma1.js

```diff
@@ -0,0 +1,17 @@
+function cutIt(arr) {
```

Comment:
we speak about these!
your code from draft is appear here!

you should delete this file

P.S.
the same with roma2.js